frontend/components/cookies/pug/cookies.pug

```diff
@@ -0,0 +1,94 @@
+#legal
```

Comment:
Should this be in here?

Reply:
Definitely no. I forgot to remove this file after testing routing of static pages.